src/components/GAddressLookup/component.js

```diff
@@ -0,0 +1,271 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import withReduxForm from '../../libs/hocs/withReduxForm';
+import Lookup from '../Lookup';
+import LocationIcon from './icons/locationIcon';
+import LocationItemIcon from './icons/locationItemIcon';
+import SelectedLocationIcon from './icons/selectedLocationIcon';
+import SearchValueIcon from './icons/searchValueIcon';
+import { uniqueId } from '../../libs/utils';
+import poweredByGoogleColored from '../../../assets/images/google/powered_by_google_on_white.png';
+import './styles.css';
+
+function formatSuggestionItem(suggestion, highlightMatch, icon) {
+    let formattedLabel;
+
+    if (highlightMatch) {
+        suggestion.structured_formatting.main_text_matched_substrings.forEach(match => {
```

Comment:
We should use reduce instead of forEach since it is not functional, here you create a formattedLabel variable and then mutate it inside forEach, so if you use reduce there is not mutation. Also try to find another solution that does not use dangerouslySetInnerHTML since it not recommended. Also don't use React.createElement.
Try to create the markup with reduce.